tests/web-server/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import { describe, it, before, after } from 'node:test';
```

Comment:
@droguljic to simplify maintenance, we could move examples to tests, since we'd like to test each component, and integration between components anyway.
What do you think?

Reply:
In my opinion, examples serve the different purpose and having them is definitely valuable. I agree that we should test each component and integration between them. 
In current state I'm not sure that examples could be just copied and run as is, if I remember correctly there was some issue when I tried to do so.
My suggestion would be to reference examples in tests and in that way we ensure that they are reproducible and on the other hand we test component and integration, while leaving them to serve their purpose.

Reply:
That makes sense, but it might also become complex to maintain. If we're looking to match examples with tests, we'll either introduce noise to examples by having redundant examples, or to tests by trying to make use of a single example to test many properties and assumptions.

We could also have examples only serve as examples, and rely on test setup to provide deployable stacks. It definitely doesn't make sense to have deployable stacks in both tests and examples, as that's a lot of surface to maintain (upgrades, testing that it works, updating API...).

Reply:
Continuing on the offline conversation, the current idea is to keep complex uses cases that test integration between many components as part of examples, and more simpler ones as part of tests.